packages/lib/actions/gotToURL/action.json

```diff
@@ -8,6 +8,13 @@
         "type": "value",
         "value": "https://example.com"
       }
+    },
+    {
+      "name": "Open in new tab",
```

Comment:
It would be more flexible if we called this parameter `target` and allowed passing one of the valid target properties? See https://developer.mozilla.org/en-US/docs/Web/API/Window/open#target

While it does mean that users will need to know what a valid `target` is, it also helps them learn the web platform ;)